smart-contracts/contracts/Migrations.sol

```diff
@@ -8,7 +8,7 @@ contract Migrations {
     if (msg.sender == owner) _;
   }
 
-  function Migrations() public {
+  constructor() public {
```

Comment:
This fails compilation on my side. Is there something I need to do (update my solc version?) if I want to have that syntax available?